### PR TITLE
fix: return same SDK instance when calling `init` multiple times [EXT-3497]

### PR DESCRIPTION
**Pull request by andipaetzold**
# Purpose of PR

With this PR we ensure that only a single SDK instance exists within an app.

This fixes an issue where multiple SDKs subscribe and respond to an event sent from the host application. E.g. the `sdk.app.onConfigure` hook.

This might be a breaking change if someone decides to mutate the returned SDK. With this change, the mutation would also be returned for all other `init` calls. I assume only very few people do this, so this is only a patch release.

## PR Checklist

- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Typescript typings are added/updated/not required

### CODE REVIEW

File: lib/initialize.ts [hunk 1]
```diff
@@ -34,26 +36,33 @@ In order for the ui-extension-sdk to function correctly, your app needs to be ru
...
+
+          // Handle pending incoming messages.
+          // APIs are created before so handlers are already
+          // registered on the channel.
+          messageQueue.forEach((m) => {
+            // TODO Expose private handleMessage method
+            ;(channel as any)._handleMessage(m)
+          })
+
+          resolve([api, customApi])
```
Comment (loweisz) [inline lib/initialize.ts:58]: I think if you just return the value in the `then`, you don't need to create wrap it with a new promise and should get the promise automatically

**Review by loweisz — APPROVED** [review]
Just a nit, otherwise lgtm

**Review by Jwhiles — APPROVED** [review]
Looks nice!

File: lib/initialize.ts (hunk 1 above)
Comment (andipaetzold) [reply lib/initialize.ts:58]: Good point. Should make the code less indented

**Pull request merged**